demo/server/keycloak.json

```diff
@@ -0,0 +1,9 @@
+{
+  "realm": "raincatcher",
+  "auth-server-url": "http://localhost:8080/auth",
+  "ssl-required": "external",
+  "resource": "raincatcher-cloud",
+  "public-client": true,
+  "use-resource-role-mappings": true,
```

Comment:
Out of the box, they will use global realms. But I was thinking if you wanted to show how a "Mobile Only" role could be created, you could show that without changing the demo apps code. That was the main reason really.